src/writer/logs.py

```
@@ -190,7 +190,11 @@ def format(self, record: logging.LogRecord, **kwargs) -> str:
             if len(record.args) > 0 and isinstance(record.args[0], dict):
                 data.update(record.args[0])
 
-        return json.dumps(data)
+        try:
+            data_as_json = json.dumps(data)
+            return data_as_json 
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)